superdesk/core/web/endpoints.py

```diff
@@ -145,7 +145,7 @@ def endpoint(
 
         def fdec(func: EndpointFunction):
             endpoint_func = Endpoint(
-                f"{self.url}/{url}" if self.url else url,
+                f"/{self.url_prefix}{url}" if self.url_prefix else url,
```

Comment:
This might play up with the RestEndpoints, as it uses `url`. We can change the RestEndpoints to use `url_prefix` instead.

Reply:
than means the `url_prefix` here is useless?